lib/ansible/modules/network/cloudengine/ce_bgp_neighbor_af.py

```diff
@@ -1095,8 +1095,8 @@ def check_bgp_neighbor_af_other(self, **kwargs):
                 else:
                     need_cfg = True
 
-        redirect_ip_vaildation = module.params['redirect_ip_vaildation']
-        if redirect_ip_vaildation != 'no_use':
+        redirect_ip_validation = module.params['redirect_ip_validation']
+        if redirect_ip_validation != 'no_use':
 
             conf_str = CE_GET_BGP_PEER_AF_HEADER % (
                 vrf_name, af_type, remote_address) + "<redirectIPVaildation></redirectIPVaildation>" + CE_GET_BGP_PEER_AF_TAIL
```

Comment:
Thank you for your attention first.
But I don't think that is an issue.
Such as line 1102 which xml tag is 'redirectIPVaildation'.
So parameter 'redirect_ip_validation' is named from there.

Reply:
do you suggest to leave the option name as it is?

Reply:
This is an interesting question.
I am sure that the word is from [NETCONF Schema API Reference](https://support.huawei.com/enterprise/en/doc/EDOC1100074782?idPath=24030814%7C21782165%7C21782239%7C22318639%7C21059820).

Reply:
i can't open that. it is a clear typo anyway, looks strange, better to inform huawei about this.
i added the aliase, is it ok?

Reply:
@xuxiaowei0512 if you agree, would be cool if you approved this :) thanks